### ISSUE TITLE
Is it possible to train lightweight GAN from scratch to reach the level of StyleGAN2/2ada transfer learning?

### ISSUE THREAD

**Issue by ilem777** [lucidrains/lightweight-gan#73]
My dataset has 9,000 256px images, and my previous study used StyleGAN2 and StyleGAN2 ADA for transfer learning, and the result after one day of training was FID 31 for the former and FID 19 for the latter, then I used your lightweight GAN for one day training from scratch, and the final result was FID 59, with overfitting during training.

The specific training command line: `lightweight_gan --data ./data --name test_1 --batch-size 4 --gradient-accumulate-every 2 --image-size 256 --optimizer adam --activation leaky_relu --aug-prob 0.25 --aug-types [color,cutout,offset,translation] --calculate_fid_every 5000 --calculate_fid_num_images 50000`

**Comment by ilem777** [comment]
@Dok11 All right, disappointment.
